internal/provider/provider.go

```diff
@@ -46,6 +46,7 @@ func New() func() *schema.Provider {
 				"hcp_hvn":                            resourceHvn(),
 				"hcp_hvn_peering_connection":         resourceHvnPeeringConnection(),
 				"hcp_hvn_route":                      resourceHvnRoute(),
+				"hcp_peering_connection_activation":  resourcePeeringConnectionActivation(),
```

Comment:
How do we feel about this name? 🤔 another idea was `hcp_peering_wait_for_active`